modules/govuk_postgresql/templates/usr/local/bin/wal-e_env_sync

```diff
@@ -29,13 +29,13 @@ else
   sudo service postgresql stop
 
   # Push the passphrase to a file so we can decrypt without prompt
-  sudo envdir <%= @env_sync_envdir %> sh -c  'echo passphrase $GPG_PASSPHRASE >> /var/lib/postgresql/.gnupg/gpg.conf'
+  sudo envdir /etc/wal-e/env_sync/env.d sh -c  'echo -n "\npassphrase ${GPG_PASSPHRASE}\n" >> /var/lib/postgresql/.gnupg/gpg.conf'
```

Comment:
When Puppet runs [this code](https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk_postgresql/manifests/wal_e/backup.pp#L132-L138) should clean up the passphrase file.